app/views/discovered_hosts/welcome.html.erb

```diff
@@ -1,10 +1,16 @@
+<% content_for(:javascripts) do %>
+  <%= webpacked_plugins_js_for :'foreman_discovery' %>
+<% end %>
+<% content_for(:stylesheets) do %>
+  <%= webpacked_plugins_css_for :'foreman_discovery' %>
+<% end %>
+
 <% content_for(:title, _("Discovered Hosts")) %>
-<div class="blank-slate-pf">
-  <div class="blank-slate-pf-icon">
-    <%= icon_text("gears", "", :kind => "fa") %>
-  </div>
-  <h1><%= _('Discovered Hosts') %></h1>
-  <p><%= _("No discovered hosts found in this context.") %>
-  <%= _("This page shows discovered bare-metal or virtual nodes waiting to be provisioned.") %></p>
-  <p><%= link_to _('Learn more about this in the documentation.'), documentation_url("#{ForemanDiscovery::VERSION.scan(/\d+\.\d+/).first}/index.html", {:root_url => 'https://theforeman.org/plugins/foreman_discovery/'})%></p>
-</div>
+
+<% content_for(:content) do %>
+  <%= notifications %>
+  <div id="organization-id" data-id="<%= Organization.current.id if Organization.current %>" ></div>
+  <div id="user-id" data-id="<%= User.current.id if User.current %>" ></div>
+  <%= react_component('DiscoveredHosts',
+       docUrl: "https://theforeman.org/plugins/foreman_discovery/#{ForemanDiscovery::VERSION.scan(/\d+\.\d+/).first}/index.html" ) %>
```

Comment:
documentation page doesn't seem to exit yet for this version, any chance we could link to a `latest` url?
cc @lzap

Reply:
IMO, the documentation link will be generated only when a new version of the plugin is released, So the current URL needs to change. Correct me if I am wrong @lzap

Reply:
Yeah it is this status quo mode - it will be only valid after we do the release and new docs. I'd waive this in at this point. I would love to fix all our docs links in Foreman core and create some unified landing page ... someday.